tests/test_model.py

```diff
@@ -0,0 +1,29 @@
+import testing_constants
+from pytorch_lightning import Trainer
+from pytorch_lightning.callbacks import LearningRateMonitor
+from pytorch_lightning.callbacks.progress import TQDMProgressBar
+from pytorch_lightning.loggers import CSVLogger
+
+from modsim2.data.loader import DMPair
+from modsim2.model.resnet import ResnetModel
+
+# Constants for testing model running and training
```

Comment:
I think we can just move these inside the test in this case, they feel non-trivial enough. If we want them available to multiple tests we can consider a fixture

Reply:
Have done this too